duplicate_code_detection.py

```diff
@@ -309,9 +347,18 @@ def run(
             if similarity_percentage < ignore_threshold:
                 continue
             short_source_path = source.replace(project_root_dir, "")
-            code_similarity[short_source_file_path][short_source_path] = round(
-                similarity_percentage, 2
-            )
+            if show_loc:
+                code_similarity[short_source_file_path][short_source_path] = dict()
+                code_similarity[short_source_file_path][short_source_path][loc_label] = get_loc_count(
+                    source_file
+                )
+                code_similarity[short_source_file_path][short_source_path][similarity_label]  = round(
+                    similarity_percentage, 2
+                )
+            else:
+                code_similarity[short_source_file_path][short_source_path] = round(
+                    similarity_percentage, 2
+                )
```

Comment:
What if we keep the same structure in both cases but in the "LOC" case the `loc_label` key is empty?

I might have misunderstood something, but while it seems more "complex" data-structure wise, it's simpler code-wise.

Reply:
sure, but it will change the default behavior because the json output won't have the same structure any more. So it's up to you.

Reply:
Hmmm I see. Let's have it as you propose in your pull request for now.